_sass/customize_theme.scss

```diff
@@ -103,11 +103,13 @@ table.navbar-brand {
     height: 32px;
     font-size: 28px;
     line-height: 28px;
+    cursor: default;
```

Comment:
Needs to be `cursor: pointer;`, and need it also for the image, so easiest is to just add this AFTER the .navbar-brand-subtitle css:

```
.navbar-brand * {
    cursor: pointer;
}
```